<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManage</class>
 <widget class="QDialog" name="UserManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>添加(&amp;A)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Delete_Button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>删除(&amp;D)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Allpick_Button">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>全选(&amp;L)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
